### PR TITLE
Feature/signup

### PR DESCRIPTION
**Pull request by xlasserre**
As a user I should be able to manually create an account

https://trello.com/c/wOuHs7O8/2-5-as-a-user-i-should-be-able-to-manually-create-an-account

simulator screen shot - iphone 6 - 2018-06-04 at 18 41 22

### CODE REVIEW

File: src/components/common/CustomPicker/styles.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    alignSelf: 'center'
+  },
+
+  pickerStyle: {
+    fontSize: 16,
+    paddingTop: 13,
+    paddingHorizontal: 10,
+    paddingBottom: 12,
+    borderWidth: 1,
+    borderColor: 'gray',
```
Comment (agch-dev) [inline src/components/common/CustomPicker/styles.js]: Put `'gray'`, `'white'` and  `'black'` as constants in your stylesConstants file with their hex values if you haven't already and import them here.

File: src/containers/SignUpScreen/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { connect } from 'react-redux';
+import { KeyboardAwareView } from 'react-native-keyboard-aware-view';
+import { object, func } from 'prop-types';
+
+import SignUpForm from '../../components/SignUpForm';
+import { signUp } from '../../actions/userActions';
+import styles from './styles';
+
+class SignUpScreen extends React.Component {
+  constructor(props) {
```
Comment (agch-dev) [inline src/containers/SignUpScreen/index.js]: `props` not needed

File: src/containers/SignUpScreen/index.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    this.navigateToLogin = this.navigateToLogin.bind(this);
+  }
+
+  signUp(userData) {
+    const jsUser = userData.toJS();
+    jsUser.gender = jsUser.gender.toLowerCase();
+    const user = {
+      user: jsUser
+    };
+    this.props.userSignUp(user);
```
Comment (agch-dev) [inline src/containers/SignUpScreen/index.js]: I would simplify this code by naming the `jsUser` const just `user` then here you can just do: `this.props.userSignUp({ user });`.

File: src/containers/SignUpScreen/index.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  }
+
+  render() {
+    return (
+      <KeyboardAwareView animated>
+        <ScrollView>
+          <ImageBackground source={require('../../assets/images/groupTop.png')} style={styles.backImage}>
+            <View style={styles.content}>
+              <Text style={styles.title}>TARGET MVD</Text>
+              <View>
```
Comment (agch-dev) [inline src/containers/SignUpScreen/index.js]: is this `View` really necessary? you are not assigning any style to it and the form component inside probably has a `View` as it's outermost component. So any styling could be done there if necessary.

**Review by agch-dev — CHANGES_REQUESTED** [review]
Great work! Just a few comments.

File: src/components/common/CustomPicker/styles.js (hunk 1 above)
Comment (xlasserre) [reply src/components/common/CustomPicker/styles.js]: Yeah, totally missed that one.

File: src/containers/SignUpScreen/index.js (hunk 4 above)
Comment (xlasserre) [reply src/containers/SignUpScreen/index.js]: Yeah, you're right. I suppose I added it there to organize myself and then I didn't delete it even though it didn't have any use.

**Review by bernabe9 — APPROVED** [review]
Looks great! No comments to add

**Comment by xlasserre** [comment]
@aguscha333 I've already added the changes you requested :)

**Review by agch-dev — APPROVED** [review]

**Pull request merged**